src/chimera/interfaces/telescope.py

```diff
@@ -417,6 +417,28 @@ def unparkComplete(self):
         successfully.
         """
 
+    def openCover(self):
```

Comment:
Yeah... The idea is VERY good but Pyro does not let you do that... :-(

```python
from chimera.interfaces.telescope import Telescope
from chimera.core.manager import Manager
manager = Manager()
telescope = manager.getProxy("127.0.0.1:7666/Telescope/0")
print(type(telescope))
issubclass(telescope, Telescope)
```

gives you

```
<class 'chimera.core.proxy.Proxy'>
Traceback (most recent call last):
  File "<string>", line 6, in <module>
TypeError: issubclass() arg 1 must be a class
```

Any ideas?